src/server/game/Entities/Player/Player.cpp

```diff
@@ -15026,7 +15028,10 @@ void Player::PrepareGossipMenu(WorldObject* source, uint32 menuId /*= 0*/, bool
                         }
                         break;
                     }
+                }
+                    break;
                 case GOSSIP_OPTION_LEARNDUALSPEC:
+                    [[fallthrough]];
```

Comment:
do we need this `[[fallthrough]];` here? I thought it's only needed when there are actually some lines of code between two cases and no `break`

Reply:
You're right! For some reason I thought the compiler was giving an error in here, my bad!
I've fixed the other warnings with Fallthrough.